projects/ion/src/lib/dropdown/dropdown.component.spec.ts

```diff
@@ -497,3 +497,55 @@ describe('IonDropdownComponent / Required', () => {
     expect(document.getElementById('option-0')).toHaveClass('dropdown-item');
   });
 });
+
+describe('IonDropdownComponent / No data', () => {
+  const defaultNoData = {
+    ...defaultDropdown,
+    options: [],
+    selected: {
+      emit: selectEvent,
+    } as SafeAny,
+  };
+
+  it('Should render NoData when there are no options.', async () => {
+    await sut(defaultNoData);
+    const element = screen.getByTestId('ion-no-data');
+    expect(element).toBeInTheDocument();
+    expect(screen.getByText('Não há dados')).toBeInTheDocument();
+    expect(
+      document.getElementById('ion-icon-exclamation-rounded')
+    ).toBeInTheDocument();
+  });
+
+  it('Should not render NoData when there are options.', async () => {
+    await sut({
+      ...defaultNoData,
+      options: [{ label: 'Option 1', selected: false }],
+    });
+    const element = screen.queryByTestId('ion-no-data');
+    expect(element).not.toBeInTheDocument();
+  });
+
+  it('Should render NoData with default parameters.', async () => {
```

Comment:
This test is practically the same as the 510 line

Reply:
ok, I'm going to delete the it that is repeating the validation of the default parameters